python/lsst/cp/pipe/utils.py

```diff
@@ -282,6 +282,51 @@ def makeDataRefList(self, namespace):
             self.refList += refList
 
 
+def irlsFit(initialParams, dataX, dataY, function, weightsY=None):
```

Comment:
The two commits adding this should be squashed together so the indentations are correct on the first ticket.